docker-compose.overrides.yml

```
@@ -0,0 +1,4 @@
+services:
+  es:
+    # Memory limit for ES, as it tends to be a memory hoarder
+    mem_limit: 4294967296
```

This option is removed in newer docker compose file versions.[^1]

Specifying the value in [high-level units](https://docs.docker.com/compose/compose-file/compose-file-v3/#specifying-byte-values) is also possible if you still want to keep it.

[^1]: [Compose file reference > Legacy versions > About versions and upgrading](https://docs.docker.com/compose/compose-file/compose-versioning/#version-3)

```suggestion
    mem_limit: 4gb
```

---

I'd be happy to remove it, but I assume some people rely on this to prevent ES from sucking up their RAM? We're still on compose file version 2.4 for now: https://github.com/WordPress/openverse/blob/2ecb187efe8d2afc11b72fd4ba2dc9a2a161edea/docker-compose.yml#L1

When we update the compose format, maybe then we can remove it? I don't want to cause a disruption in this PR in the mean time.